code/modules/mob/living/carbon/human/species_types/plasmamen.dm

```diff
@@ -162,3 +162,18 @@
 		randname += " [lastname]"
 
 	return randname
+
+/datum/species/plasmaman/handle_chemicals(datum/reagent/chem, mob/living/carbon/human/H)
+    . = ..()
+
+    if(chem.type == /datum/reagent/toxin/plasma)
+        H.adjustOxyLoss(-1)
+        H.reagents.remove_reagent(chem.type, REAGENTS_METABOLISM)
+
+    if(H.reagents.has_reagent(/datum/reagent/stable_plasma))
+        H.adjustBruteLoss(-0.5)
+        H.adjustFireLoss(-0.5)
```

Comment:
Would make more sense to have plasma(the one you have to grind up) heal brute and burn instead of the one freely available at any chem dispenser

Reply:
And done.